deploy/manifests/dev/us-east-2/tenant/storetheindex/instances/ago/kustomization.yaml

```
@@ -28,3 +28,8 @@ configMapGenerator:
 
 patchesStrategicMerge:
   - deployment.yaml
+
+images:
+- name: storetheindex
+  newName: 407967248065.dkr.ecr.us-east-2.amazonaws.com/storetheindex/storetheindex # {"$imagepolicy": "storetheindex:storetheindex:name"}
+  newTag: 20230223171347-d23a8e11b899eac7ffa8661caf6108c56beb73f0 # {"$imagepolicy": "storetheindex:storetheindex:tag"}
```

if image policy comments are not removed, CD will try and update them.
```suggestion
  newName: 407967248065.dkr.ecr.us-east-2.amazonaws.com/storetheindex/storetheindex
  newTag: 20230223171347-d23a8e11b899eac7ffa8661caf6108c56beb73f0
```